apis/python/test/test_object_index.py

```diff
@@ -145,6 +145,7 @@ def test_object_index_ivf_flat(tmp_path):
     )
 
     index.update_index()
+    index = object_index.ObjectIndex(uri=index_uri)
```

Comment:
(Non-blocking) Could you explain a bit more about why we need to do this? It seems much simpler to not have to?

Reply:
We can't pass the initialized `ObjectIndex` to the taskgraph as this would need to pickle all open arrays and memory of the open index class. This is why we just pass the `index_uri` and re-initialize on the Taskgraph side.